app-emulation/qemu/qemu-6.2.0.ebuild

```diff
@@ -33,9 +33,9 @@ LICENSE="GPL-2 LGPL-2 BSD-2"
 SLOT="0"
 
 IUSE="accessibility +aio alsa bpf bzip2 capstone +caps +curl debug +doc
-	+fdt fuse glusterfs gnutls gtk infiniband iscsi io-uring
-	jack jemalloc +jpeg kernel_linux
-	kernel_FreeBSD lzo multipath
+	+fdt fuse glusterfs +gnutls gtk infiniband iscsi io-uring
```

Comment:
Starting from 6.2.0, because of the change to `+gnutls`, qemu pulls in gnutls by default, and that results in pulling in libtasn1 as well as nettle in the SDK. I am not sure I like the change, as it makes the SDK size bigger again.
However, as you wrote in the description, the `vnc` requires gnutls. If we would want to disable gnutls, we would need to disable vnc as well. In some cases vnc would be needed for debugging, so I think we should keep enabling both vnc and gnutls.

Reply:
I'm not worried about the size of SDK as much as the size of stuff installed into the boot partition in the production image for example. I mean, yes, it's annoying the SDK gets bigger, but if the SDK size is a problem, then I'm sure we have some unnecessary stuff there that could be dropped right away.